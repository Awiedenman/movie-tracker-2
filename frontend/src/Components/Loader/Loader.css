.loader {
  align-items: center;
  background: #000;
  color: var(--primary-red);
  display: flex;
  height: 100vh;
  justify-content: center;
  width: 100vw;
}