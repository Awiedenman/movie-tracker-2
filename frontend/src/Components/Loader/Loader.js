import React, { Component } from 'react';
import { BounceLoader } from 'halogenium';

import './Loader.css';

class Loader extends Component {
  render() {
    return (
      <div className="loader">
        <BounceLoader color="#e50914" size="10rem" margin="4px" />
      </div>
    );
  }
}

export default Loader;