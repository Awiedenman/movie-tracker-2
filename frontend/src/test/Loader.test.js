import React from 'react';
import { shallow } from 'enzyme';
import Loader from '../Components/Loader/Loader';

describe('Loader', () => {
  let wrapper;

  beforeEach(() => wrapper = shallow(<Loader />));

  test('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
