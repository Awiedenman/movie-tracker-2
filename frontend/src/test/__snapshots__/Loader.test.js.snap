// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader should match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Loader />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <BounceLoader
        color="#e50914"
        loading={true}
        margin="4px"
        size="10rem"
/>,
      "className": "loader",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "color": "#e50914",
        "loading": true,
        "margin": "4px",
        "size": "10rem",
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <BounceLoader
          color="#e50914"
          loading={true}
          margin="4px"
          size="10rem"
/>,
        "className": "loader",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "color": "#e50914",
          "loading": true,
          "margin": "4px",
          "size": "10rem",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
